include/crm/common/Makefile.am

```diff
@@ -47,6 +46,7 @@ noinst_HEADERS = acl_internal.h 	\
 				 health_internal.h	\
 				 internal.h		\
 				 io_internal.h		\
+				 ipc_attrd_internal.h	\
```

Comment:
You can drop this commit too, @clumens has a PR changing this to a wildcard

Reply:
Done